### PR TITLE
More Resnet Optimizations

### PR DESCRIPTION
**Pull request by tt-aho**

### CODE REVIEW

File: tt_metal/third_party/umd
Comment (tt-rkim) [inline tt_metal/third_party/umd:1]: jeez we doing this update? it's needed?

Comment (tt-aho) [reply tt_metal/third_party/umd:1]: Yessir it's needed

**Review by tt-rkim — APPROVED** [review]

**Comment by tt-rkim** [comment]
make sure perf pipelines and t3k unit tests are ok

**Review by pgkeller — APPROVED** [review]

**Review by TT-BrianLiu — APPROVED** [review]

**Review by AleksKnezevic — APPROVED** [review]

**Review by tt-nshanker — APPROVED** [review]

File: tt_metal/impl/dispatch/command_queue.cpp [hunk 1]
```diff
@@ -1299,6 +1300,7 @@ void HWCommandQueue::enqueue_read_buffer(Buffer& buffer, void* dst, bool blockin
 
     if (is_sharded(buffer.buffer_layout())) {
         bool width_split = buffer.shard_spec().shape_in_pages()[1] != buffer.shard_spec().tensor2d_shape[1];
+        std::cout << width_split << std::endl;
```
Comment (abhullar-tt) [inline tt_metal/impl/dispatch/command_queue.cpp:1303]: debug print reminder

File: tt_metal/llrt/tt_cluster.hpp [hunk 2]
```diff
@@ -94,6 +94,14 @@ class Cluster {
         return device->get_fast_pcie_static_tlb_write_callable(mmio_device_id);
     }
 
+    tt::Writer get_static_tlb_writer(tt_cxy_pair target) const {
```
Comment (abhullar-tt) [inline tt_metal/llrt/tt_cluster.hpp:97]: can you add a comment about when this should be used / scenarios where this could provide perf benefit

**Review by abhullar-tt — APPROVED** [review]

**Review by arakhmati — APPROVED** [review]

**Pull request merged**